gce_image_import_export_tests/test_suites/export/image_export_test_suite.go

```diff
@@ -127,29 +128,33 @@ func runImageExportWithRichParamsTest(ctx context.Context, testCase *junitxml.Te
 	bucketName := fmt.Sprintf("%v-test-image", testProjectConfig.TestProjectID)
 	objectName := fmt.Sprintf("e2e-export-rich-param-test-%v", suffix)
 	fileURI := fmt.Sprintf("gs://%v/%v", bucketName, objectName)
+	region, _ := paramhelper.GetRegion(testProjectConfig.TestZone)
 
 	argsMap := map[testsuiteutils.TestType][]string{
 		testsuiteutils.Wrapper: {"-client_id=e2e", fmt.Sprintf("-project=%v", testProjectConfig.TestProjectID),
-			"-source_image=e2e-test-image-10g", fmt.Sprintf("-destination_uri=%v", fileURI),
-			fmt.Sprintf("-network=%v-vpc-1", testProjectConfig.TestProjectID),
-			fmt.Sprintf("-subnet=%v-subnet-1", testProjectConfig.TestProjectID),
+			"-source_image=global/images/e2e-test-image-10g", fmt.Sprintf("-destination_uri=%v", fileURI),
+			fmt.Sprintf("-network=global/networks/%v-vpc-1", testProjectConfig.TestProjectID),
+			fmt.Sprintf("-subnet=projects/%v/regions/%v/subnetworks/%v-subnet-1",
+				testProjectConfig.TestProjectID, region, testProjectConfig.TestProjectID),
```

Comment:
We should still test network/subnet name-only scenarios instead of replacing them with full path scenarios. Both are needed. Applies for tests below as well

Reply:
A name-only subnetwork param is covered by another test "runImageExportWithSubnetWithoutNetworkParamsTest".
I was trying to cover these scenarios by as less e2e tests as possible because e2e is expensive. I won't test all combination of them.

Please let me know if you don't agree.